packages/core/src/combinator/zip.js

```diff
@@ -7,20 +7,20 @@ import { empty } from '../source/core'
 import Pipe from '../sink/Pipe'
 import IndexSink from '../sink/IndexSink'
 import { disposeAll } from '@most/disposable'
-import { map as mapArray, tail } from '@most/prelude'
+import { map as mapArray } from '@most/prelude'
 import invoke from '../invoke'
 import Queue from '../Queue'
 
 /**
- * Combine streams pairwise (or tuple-wise) by index by applying f to values
- * at corresponding indices.  The returned stream ends when any of the input
- * streams ends.
+ * Combine two streams pairwise (or tuple-wise) by index by applying f to
+ * values at corresponding indices.  The returned stream ends when any of
```

Comment:
We can be more specific here now: "The returned stream ends with either of" (any -> either)